docker-compose.example.yml

```diff
@@ -7,7 +8,10 @@ services:
     restart: unless-stopped
     volumes:
       - ./server/src/configs/areas.json:/home/node/server/src/configs/areas.json
-      - ./server/src/configs/local.json:/home/node/server/src/local/config.json
+      - ./server/src/configs/local.json:/home/node/server/src/local/local.json
+      - ./geofence/:/home/node/public/geofence
```

Comment:
Why are we copying the geofence into the public folder? It's not used there...

Reply:
Will push update for it.